s3-proxy/src/client_impls/azure.rs

```
@@ -164,6 +168,15 @@ impl ObjectStoreClient for AzureObjectStoreClient {
         }
     }
 
+    async fn put_bucket_versioning(
+        &self,
+        req: S3Request<PutBucketVersioningInput>,
+    ) -> S3Result<S3Response<PutBucketVersioningOutput>> {
+        println!("put_bucket_versioning: {:?}", req);
```

Remove these, panic or error instead of print statement and return OK? never use this function for azure, just assume it always works